test/src/matomo_test.dart

```diff
@@ -50,6 +50,7 @@ void main() {
           (tracker, fixedDateTime) =>
               expect(tracker.session.lastVisit, fixedDateTime.toUtc()),
           (tracker, _) => expect(tracker.session.visitCount, 1),
+          (tracker, _) => expect(tracker.customHeaders, customHeaders),
```

Comment:
Is this in the "it should be able to initialize" test?
I think it would be better to have a separated specific test.

Reply:
Yes, and test that it works with and without custom header